backend/api/serializers/TimeRecordSerializer.py

```diff
@@ -42,9 +43,10 @@ class Meta:
         model = TimeRecord
         fields = ('id', 'athlete', 'athlete_full_name','event_type', 'event_type_name', 'swim_meet', 'time', 'date')
 
+    
     def validate_time(self, value):
-        # validation for time format MM:SS.mm
-        if not re.match(r'^([0-5]?\d:)?([0-5]?\d\.\d{1,2})$', value):
-
-            raise serializers.ValidationError("Invalid time format. Use [M:]S.mm")
+        # timedelta object with zero duration for comparasion purposes
+        zero_duration = timedelta()
+        if value < zero_duration:
```

Comment:
Shouldn't be <= zero_duration? do we want to save a time of 0? Also, right now "" is save as 0, should we do something about that?
What do you think?

Reply:
Yeah, it doesn't make sense to allow a value of 0. I change the condition to <=. 
This also takes care of "", since it was saved as 0, is now not allowed.